packages/core/src/panel/Panel.tsx

```diff
@@ -16,14 +16,28 @@ import "./Panel.css";
  *
  */
 
+export interface PanelProps extends HTMLAttributes<HTMLDivElement> {
+  /**
+   * Emphasis styling variant; defaults to "medium".
+   */
+  emphasis?: "medium" | "high";
+}
+
 const withBaseName = makePrefixer("uitkPanel");
 
-export const Panel = forwardRef<HTMLDivElement, HTMLAttributes<HTMLDivElement>>(
-  function Panel({ className, children, ...restProps }, ref) {
-    return (
-      <div className={cx(withBaseName(), className)} ref={ref} {...restProps}>
-        {children}
-      </div>
-    );
-  }
-);
+export const Panel = forwardRef<HTMLDivElement, PanelProps>(function Panel(
+  { className, children, emphasis = "medium", ...restProps },
+  ref
+) {
+  return (
+    <div
+      className={cx(withBaseName(), className, {
+        uitkEmphasisHigh: emphasis === "high",
```

Comment:
I think i'd prefer to not use a generic class for emphasis. 
Imagine a situation where we have a component inside Panel which has a different emphasis, could there be any style overrides which could cause issues?

Reply:
emphasis is always attached to the component className specifically (e.g. .uitkEmphasisHigh.uitkPanel in this case) so nesting issues shouldn't occur